plugins/modules/s3_logging.py

```diff
@@ -34,6 +34,11 @@
       - "The prefix that should be prepended to the generated log files written to the target_bucket."
     default: ""
     type: str
+  acl:
+    description:
+      - "Setup target bucket ACLs to grant AWS special log delivery account to write server access logs."
```

Comment:
Since setting this to False removes the ACL I think we should add a note to that effect here...